docs/src/introduction/getting-started.md

```diff
@@ -1 +1,131 @@
-> [WIP]
+# Getting started
+
+This tutorial guides you through the process of connecting to a remote Miden-Node using the Miden client. The Miden node; that is, the software which processes transactions and creates blocks for the Miden rollup and the Miden client, which provides a way to execute and prove transactions, facilitating the interaction with the Miden rollup. By the end of this tutorial, you will be able to configure the Miden client, connect to a Miden-Node, and perform basic operations like sending transactions, generating and consuming notes.
+
+## Prerequisites
+
+Before starting, ensure you have the following:
+
+- **Rust Installed:** You must have the Rust programming language installed on your machine. If you haven't installed Rust, you can download it from [the Rust website](https://www.rust-lang.org/learn/get-started).
+- **Node IP Address:** Obtain the IP address of the running Miden-Node. This information can be acquired by contacting one of the Miden engineers.
+- **Miden-Client Installation:** You need to install the [Miden client](https://github.com/0xPolygonMiden/miden-client) and configure it to point to the remote node.
+
+## Step 1: Configuring the Miden-Client
+
+1. **Download the Miden-Client:** First, download the Miden client from its repository. Use the following command:
+
+   ```shell
+   git clone https://github.com/0xPolygonMiden/miden-client
+   ```
+
+2. **Navigate & Configure the Client:** Navigate to the client directory and modify the configuration file to point to the remote Miden-Node. You can find the configuration file at `./miden.toml`. In the `[RPC]` section replace the `endpoint = { host: }` field with the address provided by the Miden engineer.
+
+   ```shell
+   cd miden-client
+   ```
+
+   Configuration file example:
+
+   ```toml
+    [rpc]
+    endpoint = { protocol = "http", host = "<NODE_IP_ADDRESS>", port = 57291 }
+
+    [store]
+    database_filepath = "store.sqlite3"
+   ```
+
+3. **Build & install the Client:** install the client using cargo:
+
+   ```shell
+   cargo install --features testing --path .
+   ```
+
+   you should now be able to use the following command:
+
+   ```shell
+   miden-client --help
+   ```
+
+## Step 2: Setting-up the Miden-Client
+
```

Comment:
Can we maybe already deploy a faucet account on the remote node? Then the user just needs to create account A and account B. That might be easier and more intuitive.

Reply:
The faucet account should already exist on the remote node - but users won't be able to get assets from it as executing transactions against the faucet account requires knowing its state and performing authorization.

One potential way to address it, is to have a simple website where the users could go, enter their account ID, and ask the faucet account to send them some tokens. Maybe this can even be run on the same machine as the node. Should we create an issue for this?

Reply:
Issue created here: https://github.com/0xPolygonMiden/miden-node/issues/215